### PR TITLE
[WIP] Feature/prereg/file picker questions

### PR DESCRIPTION
**Pull request by laurenbarker**
Throws error when destroy is called on an another question's filesWidget.

### CODE REVIEW

File: website/project/metadata/prereg-prize-test.json [hunk 1]
```diff
@@ -178,51 +268,111 @@
...
+                "help": "If any of the three indices of tastiness are not completed by a subject, that subject will not be included in the analysis.",
+                "properties": {
+                    "question": {
+                        "id": "q19a",
+                        "type": "string",
+                        "format": "textarea"
+                    },
+                    "uploader": {
+                        "id": "q19b",
+                        "type": "osf-upload"
```
Comment (samchrisinger) [inline website/project/metadata/prereg-prize-test.json:337]: @leb2dg What do you think about adding a format field to the osf-upload type?

Then we could have a version where the filepicker is always visible (Script) and one that is toggleable for the optional uploads

File: website/static/js/filesWidget.js [hunk 2]
```diff
@@ -23,7 +23,6 @@ var FilesWidget = oop.defclass({
             filesData: filesUrl,
             uploads: true,
             showFilter: true,
```
Comment (samchrisinger) [inline website/static/js/filesWidget.js:25]: @leb2dg is this what broke the width of the filepicker?

File: website/static/js/registrationEditorExtensions.js [hunk 3]
```diff
@@ -23,16 +23,17 @@ ko.bindingHandlers.osfUploader = {
                     this.path = item.data.path;
                     self.files = item.data;
 
+                    console.log(viewModel);
```
Comment (samchrisinger) [inline website/static/js/registrationEditorExtensions.js]: Please remove console.logs

File: website/project/metadata/prereg-prize-test.json (hunk 1 above)
Comment (laurenbarker) [inline website/project/metadata/prereg-prize-test.json:337]: That would probably be a good choice. Should I add that on to this PR or would you add that after this is merged?

Comment (samchrisinger) [inline website/project/metadata/prereg-prize-test.json:337]: Please add these changes to the PR

File: website/static/js/filesWidget.js (hunk 2 above)
Comment (laurenbarker) [inline website/static/js/filesWidget.js:25]: Why would that break the width?

Comment (samchrisinger) [inline website/static/js/filesWidget.js:25]: location: 'dashboard' was deleted here

Comment (laurenbarker) [inline website/static/js/filesWidget.js:25]: Oh, no that doesn't break the width. It actually doesn't do anything at all (according to Caner) so I removed it.

**Pull request merged**